gradlew

```diff
@@ -129,6 +130,7 @@ fi
 if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
     APP_HOME=`cygpath --path --mixed "$APP_HOME"`
     CLASSPATH=`cygpath --path --mixed "$CLASSPATH"`
+
```

Comment:
Was this newline added intentionally?

Reply:
I did not intentionally add an additional line, it may have been due to updating the version of gradle the project is using

Reply:
You're right, looks like there's now a few extra newlines that come in with the newer version of gradle.